connexion/lifecycle.py

```diff
@@ -34,8 +38,39 @@ def __init__(self,
                  content_type=None,
                  body=None,
                  headers=None):
+        if not isinstance(status_code, int) or not (100 <= status_code <= 505):
+            raise ValueError("{} is not a valid status code".format(status_code))
```

Comment:
Should connexion really be policing which status codes the user is allowed to send?